File: support_tools/swigwin-3.0.12/Examples/test-suite/ruby/static_const_member_2_runme.rb
#!/usr/bin/env ruby
#
# Put description here
#
# 
# 
# 
#

require 'swig_assert'

require 'static_const_member_2'

include Static_const_member_2

c = Test_int.new
a = Test_int::Forward_field	# should be available as a class constant
a = Test_int::Current_profile	# should be available as a class constant
a = Test_int::RightIndex	# should be available as a class constant
a = Test_int::Backward_field	# should be available as a class constant
a = Test_int::LeftIndex		# should be available as a class constant
a = Test_int.cavity_flags

File: support_tools/swigwin-3.0.12/Examples/test-suite/python/static_const_member_2_runme.py
from static_const_member_2 import *

c = Test_int()
try:
    a = c.forward_field
    a = c.current_profile
    a = c.RightIndex
    a = Test_int.backward_field
    a = Test_int.LeftIndex
    a = Test_int.cavity_flags
except:
    raise RuntimeError


if Foo.BAZ.val != 2 * Foo.BAR.val:
    raise RuntimeError
